en/Data Structures/Stacks/stack.md

```diff
@@ -14,12 +14,12 @@ peek(): Gets the value of the top element without removing it
 
 
 Working with Stacks
- 1)A pointer called TOP is used to keep track of the top element in the stack.
- 2)When initializing the stack, we set its value to -1 so that we can check if the stack is empty by comparing TOP == -1.
- 3)On pushing an element, we increase the value of TOP and place the new element in the position pointed to by TOP.
- 4)On popping an element, we return the element pointed to by TOP and reduce its value.
- 5)Before pushing, we check if the stack is already full
- 6)Before popping, we check if the stack is already empty
+ 1) A pointer called TOP is used to keep track of the top element in the stack.
```

Comment:
well now it's not a Markdown list anymore, is it? is this intended?

Reply:
yea it is intended

Reply:
this may cause poor line wrapping though